### ISSUE TITLE
Setup connection screen

### ISSUE THREAD

**Issue by fuzzyweapon** [watch-the-skies/watch-the-skies#6]
https://github.com/evennia/evennia/wiki/Connection%20Screen

**Comment by fuzzyweapon** [comment]
@HeroicLeigh :trollface: I need you to design a connect screen. ;)
Or get a minion to do it.  I've compiled some tools that could be used below.

First, I recommend using a figlet font for the title:
 https://www.google.com/webhp?sourceid=chrome-instant&ion=1&espv=2&ie=UTF-8#q=ascii%20figlet

Then for any graphical stuff I would probably use http://www.jave.de/ since it's pretty simple and free.

As a side note, when I was looking for these links, I found the following gems...
http://monodraw.helftone.com/
http://www.gridsagegames.com/rexpaint/
But they look more complicated and might be useful for designing in ascii art based stuff to give the world some fun life (like a sign hanging in a bar at the station or other things).
